### ISSUE TITLE
Multiple css files

### ISSUE THREAD

**Issue by 2560p** [ZsomborHajzer/Project-DAM#14]
Is it not beneficial to store multiple css files?
As the way the header can be implemented in, it can automatically load the needed css file.
It will also remove quite a bit of a confusion and help with resolving merge conflicts (as for example - multiple pages have got `main` element, every one of them has different properties)

**Comment by ZsomborHajzer** [comment]
For now I say we keep it separate but in one meeting we can merge the css files into one. I feel like we will have merge issues with the names of classes and ID's.

**Comment by AlvesBernardo** [comment]
Yeah let's just do mutiple for now for v2 of the main dev. And then later one we can discuss if we want to have one or mutiple Css sheets.
